main/streams/plain_wrapper.c

```diff
@@ -1414,10 +1414,9 @@ static int php_plain_files_mkdir(php_stream_wrapper *wrapper, const char *dir, i
 	}
 	while (true) {
 		int ret = VCWD_MKDIR(buf, (mode_t) mode);
-		if (ret < 0) {
-			if (options & REPORT_ERRORS) {
-				php_error_docref(NULL, E_WARNING, "%s", strerror(errno));
-			}
+		if (ret < 0 && errno != EEXIST) {
+		    /* stop running and issue a warning to client when ret < 0 and errno != EEXIST */
```

Comment:
Can you please keep the `options & REPORT_ERRORS` check here?

Reply:
And could you please switch the indentation from spaces -> tabs?

Reply:
> 
> 
> Can you please keep the `options & REPORT_ERRORS` check here?

Fix it. Thanks foy your code review.

Reply:
> 
> 
> And could you please switch the indentation from spaces -> tabs?

Fix it. Thanks foy your code review.